Hymns/Display/Song Info/SongInfoDialog.swift

```diff
@@ -13,15 +23,28 @@ struct SongInfoDialog: View {
             return ErrorView().eraseToAnyView()
         }
 
-        return VStack(alignment: .leading, spacing: 15) {
-            ForEach(viewModel.songInfo, id: \.self) { songInfo in
-                SongInfoView(viewModel: songInfo)
+        if largeSizeCategories.contains(sizeCategory) {
+            return ScrollView {
+                VStack(alignment: .leading) {
+                    ForEach(viewModel.songInfo, id: \.self) { songInfo in
+                        SongInfoView(viewModel: songInfo)
+                    }
+                }
+            }.padding()
+                .cornerRadius(5)
+                .background(Color(.secondarySystemBackground))
+                .eraseToAnyView()
+        } else {
+            return VStack(alignment: .leading, spacing: 15) {
+                ForEach(viewModel.songInfo, id: \.self) { songInfo in
+                    SongInfoView(viewModel: songInfo)
+                }
             }
+            .padding()
+            .cornerRadius(5)
+            .background(Color(.secondarySystemBackground))
+            .eraseToAnyView()
```

Comment:
Instead of doing an alternate layout here, can you just put it in a scrollview regardless of size? I don't think it hurts to be in a scroll view if scrolling is not necessary